### PR TITLE
apt_rpm: support installing a list of packages (Fixes #143)

### LINKED ISSUE #143: Package module pass list of packages to apt_rpm that is converted to string as is and not supported
##### SUMMARY

apt_rpm does not support install packages provided as list, only as comma delimited string. So package module does not work with apt_rpm
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

apt_rpm
##### ANSIBLE VERSION

```paste below
ansible 2.8.8

```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

Use simple playbook that install list of packages on Altlinux

```yaml
- name: install packages
  package:
    name:
        - nginx
        - vim
```

##### EXPECTED RESULTS

Packages succesfully installed by invoking apt_rpm

##### ACTUAL RESULTS

```paste below
TASK [install_packages : install packages] *************************************
fatal: [host-1]: FAILED! => changed=false
  msg: |-
    'apt-get -y install  'vim']' failed: E: Regex compilation error - Unmatched [ or [^
```
It is because:
```
[WARNING]: The value ['nginx', 'vim'] (type list) in a string field was converted to u"['nginx', 'librdkafka']" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
```

### PR DESCRIPTION
**Pull request by amleshkov**
##### SUMMARY
apt_rpm does not support install packages provided as list, only as comma delimited string. So package module does not work with apt_rpm

Fixes #143 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

apt_rpm
##### ADDITIONAL INFORMATION

```paste below

```

edits by Gundalow: Update summary and title

### CODE REVIEW

**Comment by felixfontein** [comment]
This is missing an important change from #144 (which I guess this PR replaces?). Also, you need a changelog fragment.

**Comment by amleshkov** [comment]
> This is missing an important change from #144 (which I guess this PR replaces?). Also, you need a changelog fragment.

Yes, it is replace for #144. Changelog fragment has been added.
https://github.com/ansible-collections/community.general/pull/169/commits/02011c7423c3fa54093459958c5ae3b6c13e0e3e

File: changelogs/fragments/apt_rpm_typefix.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - Fix package type from str to list to fix package->apt_rpm invoking with list of packages
```
Comment (felixfontein) [inline changelogs/fragments/apt_rpm_typefix.yml:2]:
```suggestion
    - apt_rpm - Fix ``package`` type from ``str`` to ``list`` to fix invoking with list of packages
```

File: plugins/modules/packaging/os/apt_rpm.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def main():
         argument_spec=dict(
             state=dict(type='str', default='installed', choices=['absent', 'installed', 'present', 'removed']),
             update_cache=dict(type='bool', default=False, aliases=['update-cache']),
-            package=dict(type='str', required=True, aliases=['name', 'pkg']),
+            package=dict(type='list', required=True, aliases=['name', 'pkg']),
```
Comment (felixfontein) [inline plugins/modules/packaging/os/apt_rpm.py:153]:
```suggestion
            package=dict(type='list', elements='str', required=True, aliases=['name', 'pkg']),
```
You also need to update the documentation of the option; add `type: list` and `elements: str` to https://github.com/ansible-collections/community.general/blob/02011c7423c3fa54093459958c5ae3b6c13e0e3e/plugins/modules/packaging/os/apt_rpm.py#L24-L27

Comment (amleshkov) [reply plugins/modules/packaging/os/apt_rpm.py]: Thanks for the tips. I am a newbie and I don’t have everything right yet

Comment (gundalow) [reply plugins/modules/packaging/os/apt_rpm.py]: You are doing well :)

**Comment by gundalow** [comment]
recheck

**Comment by amleshkov** [comment]
> recheck

Done

File: changelogs/fragments/apt_rpm_typefix.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - apt_rpm - Fix ``package`` type from ``str`` to ``list`` to fix invoking with list of packages
```
Comment (felixfontein) [inline changelogs/fragments/apt_rpm_typefix.yml:2]:
```suggestion
  - apt_rpm - fix ``package`` type from ``str`` to ``list`` to fix invoking with list of packages (https://github.com/ansible-collections/community.general/pull/169).
```

**Pull request merged**